### ISSUE TITLE
Accessibility Violations in Codemirror Component - Commented Code, disabled state and placeholder

### ISSUE THREAD

**Issue by srikant-ch5** [elyra-ai/canvas#2494]
Ref: 4038

![Image](https://github.com/user-attachments/assets/2a6c43f0-2e7d-4e6d-8717-c290dbe696af)

![Image](https://github.com/user-attachments/assets/5911691b-22a5-4add-8fbf-ff9606b134d8)

<img width="867" alt="Image" src="https://github.com/user-attachments/assets/89151ba3-aa5f-4a69-86b6-d09e48617512" />
